projects/batfish/src/org/batfish/grammar/cisco/CiscoControlPlaneExtractor.java

```diff
@@ -2440,6 +2440,14 @@ public void exitNexus_vrf_rb_stanza(Nexus_vrf_rb_stanzaContext ctx) {
    }
 
    @Override
+   public void exitNo_ip_prefix_list_stanza(No_ip_prefix_list_stanzaContext ctx) {
+      String prefixListName = ctx.name.getText();
+      if (_configuration.getPrefixLists().containsKey(prefixListName)) {
+         _configuration.getRouteMaps().remove(prefixListName);
```

Comment:
_configuration.getRouteMaps().remove(prefixListName);
SHOULD BE:
configuration.getPrefixLists().remove(prefixListName);